sdk/storage/Azure.Storage.Blobs/src/Sas/BlobAccountSasPermissions.cs

```diff
@@ -113,50 +90,52 @@ public static BlobAccountSasPermissions Parse(string s)
         /// </summary>
         /// <param name="obj">The instance to compare to.</param>
         /// <returns>True if they're equal, false otherwise.</returns>
-        public override bool Equals(object obj)
-            => obj is BlobAccountSasPermissions other && this.Equals(other);
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override bool Equals(object obj) =>
+            obj is BlobAccountSasPermissions other && this.Equals(other);
 
         /// <summary>
         /// Get a hash code for the BlobAccountSasPermissions.
         /// </summary>
         /// <returns>Hash code for the BlobAccountSasPermissions.</returns>
-        public override int GetHashCode()
-            => (this.Read ? 0b000001 : 0)
-             + (this.Add ? 0b000010 : 0)
-             + (this.Create ? 0b000100 : 0)
-             + (this.Write ? 0b001000 : 0)
-             + (this.Delete ? 0b010000 : 0)
-             + (this.List ? 0b100000 : 0)
-            ;
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override int GetHashCode() =>
```

Comment:
FYI we have `HashcodeBuilder` in Azure.Core. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/Shared/HashCodeBuilder.cs
i.e. implementation https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.ApplicationModel.Configuration/src/ConfigurationSetting.cs#L123

Reply:
Filed #6735